servlets/src/main/java/com/example/servlet/servlet/LoginServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.example.servlet.servlet;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+
+import java.io.IOException;
+
+@WebServlet(name = "login", urlPatterns = "/login")
+public class LoginServlet extends HttpServlet {
+    private String username;
+    private String password;
+
+
+    @Override
+    public void init() {
+        username= getServletContext().getInitParameter("username");
+        password = getServletContext().getInitParameter("password");
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        req.getRequestDispatcher("WEB-INF/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+//        final String hiddenInput = (String) req.getAttribute("hiddenInput");
+//        System.out.println(hiddenInput);
+
+        // final String login = (String) req.getAttribute("login");
+        // final String pass = (String) req.getAttribute("pass");
+        String login =  req.getParameter("login");
+        String pass =  req.getParameter("pass");
+        System.out.println(pass);
```

Comment:
Password logging is not a good idea for ready product (only for local debugging), remove that before submitting review